query-connector/src/app/queryBuilding/utils.ts

```diff
@@ -6,6 +9,25 @@ export type CategoryToConditionArrayMap = {
   [categoryName: string]: ConditionIdToNameMap[];
 };
 
+export type ConditionIdToValueSetArray = {
+  [conditionId: string]: ValueSet[];
+};
+
+export type ValueSetsByGroup = {
```

Comment:
is this currently working? I only have conditions

Reply:
should be; there are a handful of conditions that don't have all 3 groups (ex. Newborn screening only has labs) - so only those that have value sets in that category will show up. can look at our sync later tho!

Reply:
oops you right lol, it's working for others that have all 3, like cancer. thanks!